FAQ.md

```diff
@@ -206,6 +206,33 @@ We'd love to consume machine-readable numbering plan data (assigned ranges,
 carrier & geo mappings). If you can connect us with partners in the industry
 to achieve this, please do so. Thanks!
 
+### Why don't you support Bouvet Island (BV), Pitcairn Island (PN), Antarctica (AQ) etc.?
+
+We only support a country if:
+
+*   The country has a single country calling code. For instance, Kosovo (XK) has
```

Comment:
Maybe put the first sentence / part of each bullet into bold.
*   *The country has a single country calling code.*
*   *The country still exists.*
*   *The country has some phone numbers in use that can be ascribed to it.*
*   *It has an assigned region code.*